FirebaseAppDistribution/Sources/FIRAppDistributionUIService.m

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FirebaseAppDistribution/Sources/FIRAppDistributionUIService.h"
+#import "FirebaseAppDistribution/Sources/FIRFADLogger.h"
+#import "FirebaseAppDistribution/Sources/Public/FIRAppDistribution.h"
+#import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
+
+#import <AuthenticationServices/AuthenticationServices.h>
+#import <SafariServices/SafariServices.h>
+#import <UIKit/UIKit.h>
+
+@implementation FIRAppDistributionUIService
+
+API_AVAILABLE(ios(9.0))
```

Comment:
Wondering if ios 9 support is worth it versus starting with 10 or 11?

Firebase is moving to 10 in the fall.

Reply:
Supporting 11.x would allow us to drop support for SFSafariViewController, but we would still need to support it with iOS 10. 

Being that our product is generally used by beta testers for older iOS versions and that dropping support for iOS 9 doesn't buy us much, we'd like to ship with iOS 9 support until Firebase officially moves to iOS 10.